HW/aboutme.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About Me</title>
+</head>
+
+<body>
+    <link rel="stylesheet" href="styleaboutme.css">
+    <h2>
+        <a href="Aboutme.html">About Me</a> | <a href="Portfolio.html">Portfollo</a>
```

Comment:
when naming documents, it is best practice not to use capital letters. "about_me.html" would be better